Libraries/Opc.Ua.Configuration/ApplicationInstance.cs

```diff
@@ -177,7 +182,7 @@ public async Task Start(ServerBase server)
 
             if (m_applicationConfiguration == null)
             {
-                await LoadApplicationConfiguration(false);
+                await LoadApplicationConfiguration(false).ConfigureAwait(false);
```

Comment:
Method name should end with "Async"

Reply:
the async suffix is not enforced in this codebase (yet). Also for back compat such a API change needs to be carefully considered. There is are some improvements for async in the pipeline, when they are ready it might be a good time to also apply breaking async name changes to other API..